src/configuration/app-containers.md

```diff
@@ -521,7 +519,17 @@ crons:
         cmd: 'cd web ; drush queue-run aggregator_feeds'
 ```
 
-Also note that all server times are in UTC, and hence so are the cron schedules.
+> ** Cron notes **
```

Comment:
I don't think we even need to put this in a notes box.  It's just more info about cron, let it be prose.